Apps/ClinicalDocument/src/Models/ClinicalDocumentRecord.cs

```diff
@@ -24,15 +24,15 @@ namespace HealthGateway.ClinicalDocument.Models
     public class ClinicalDocumentRecord
     {
         /// <summary>
-        /// Gets or sets the ID.
+        /// Gets or sets the id.
         /// </summary>
-        [JsonPropertyName("healthDataId")]
+        [JsonPropertyName("id")]
```

Comment:
Nit: I don't think we typically use mapping annotations for our models.

Reply:
Many of them do, the ones I used as examples. Is this something we are avoiding now?

Reply:
I think Stephen is referring to the models used solely to transfer data between our front-end and back-end. In which case, that's on me for putting them in there and not noticing.

Reply:
Okay I will remove those.